toml-to-ical-bin/src/main.rs

```diff
@@ -102,7 +111,13 @@ impl io::Write for Output {
 fn generate() -> Result<(), Error> {
     let opts = Opt::parse();
 
-    let calendar = Calendar::load::<Session>(&opts.input)?;
+    if !opts.input.is_file() {
+        eprintln!("ERROR: Input {:?} is not a file.", opts.input);
```

Comment:
Can you instead return a new variant of `Error`?

Reply:
OK.  Done.